platformio.ini

```diff
@@ -157,10 +157,10 @@ lib_deps =
 
   https://github.com/boschsensortec/Bosch-BSEC2-Library#v1.7.2502
   boschsensortec/BME68x Sensor Library@^1.1.40407
-  https://github.com/KodinLanewave/INA3221@^1.0.0
+  https://github.com/KodinLanewave/INA3221@^1.0.1
   lewisxhe/SensorLib@0.2.0
   mprograms/QMC5883LCompass@^1.2.0
   
 
   https://github.com/meshtastic/DFRobot_LarkWeatherStation#dee914270dc7cb3e43fbf034edd85a63a16a12ee
-  https://github.com/gjelsoe/STK8xxx-Accelerometer.git#v0.1.1
```

Comment:
Uhhhh... no idea why the diff thinks I changed the last line; used the web editor in github, maybe it converted unix/windows end-of-line characters or something?